app/src/main/java/org/mozilla/focus/utils/FirebaseHelper.java

```diff
@@ -212,6 +214,7 @@ protected Void doInBackground(Void... voids) {
         map.put(FirebaseHelper.RATE_APP_DIALOG_THRESHOLD, DialogUtils.APP_CREATE_THRESHOLD_FOR_RATE_DIALOG);
         map.put(FirebaseHelper.RATE_APP_NOTIFICATION_THRESHOLD, DialogUtils.APP_CREATE_THRESHOLD_FOR_RATE_NOTIFICATION);
         map.put(FirebaseHelper.SHARE_APP_DIALOG_THRESHOLD, DialogUtils.APP_CREATE_THRESHOLD_FOR_SHARE_DIALOG);
+        map.put(FirebaseHelper.ENABLE_MY_SHOT_UNREAD, MainActivity.ENABLE_MY_SHOT_UNREAD_DEFAULT);
```

Comment:
Maybe put ENABLE_MY_SHOT_UNREAD_DEFAULT in FirebaseHelper

Reply:
Do you think I should also move the DialogUtils. ones?

Reply:
Maybe I'll move them in the future.